### ISSUE TITLE
extra/qtcreator requires libsystemd

### ISSUE THREAD

**Issue by xiannox** [artix-linux/packages#16]
I recently did a clean install of artix, and noticed that QtCreator requires libsystemd to load its plugins. I fixed the pkgbuild by replacing CONFIG+=journald with CONFIG+=syslog.

~~~sh
$ diff PKGBUILD PKGBUILD.orig 
51c51
< qmake LLVM_INSTALL_DIR=/usr QBS_INSTALL_DIR=/usr CONFIG+=syslog QMAKE_CFLAGS_ISYSTEM=-I \
---
> qmake LLVM_INSTALL_DIR=/usr QBS_INSTALL_DIR=/usr CONFIG+=journald QMAKE_CFLAGS_ISYSTEM=-I \
~~~

**Comment by alium** [comment]
Thanks for the report, I'm preparing the syslog's version for artix: qtcreator-4.7.0-1.1

**Comment by alium** [comment]
fixed in qtcreator-4.7.0-1.1
